en/docs/security-announcements/cve-justifications/2025/CVE-2023-52428.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "CVE-2023-52428"
+category: security-announcements
+date: Feb 12 2025
+---
+
+# CVE-2023-52428
+
+<p class="doc-info">WSO2 Products impacted: no</p>
+<p class="doc-info">Customers actions required: no</p>
+---
+
+### REPORTED VULNERABILITY
+
+An attacker can cause a denial of service (resource consumption) via a large JWE p2c header value (aka iteration count) for the PasswordBasedDecrypter (PBKDF2) component
+
+### REPORTED PRODUCTS
+
+- WSO2 Identity Server : 5.9.0, 5.10.0, 5.11.0, 6.0.0, 6.1.0, 7.0.0, 7.1.0
+
+### WSO2 JUSTIFICATION
+
+The WSO2 team has carried out the investigation against the identified vulnerability. We are not using the vulnerable component PasswordBasedDecrypter for decryption in WSO2 products. However, we are actively working on the required mitigation steps and updating the Nimbus JOSE+JWT version to non-vulnerable version.
```

Comment:
Can you add a bit more context on why we cannot simply issue an update here and why we have to publish a justification. Similar to how we explained the limitations we have at: 
https://security.docs.wso2.com/en/latest/security-announcements/cve-justifications/2024/CVE-2022-45868/
https://security.docs.wso2.com/en/latest/security-announcements/cve-justifications/2024/CVE-2021-23463/

Reply:
resolved with https://github.com/wso2/docs-security/pull/87/commits/6d0358f030f49dc5a37b43703fab959ae80b3df3